src/kili/cli/project/create.py

```diff
@@ -75,13 +75,9 @@ def create_project(
             json_interface = json.load(interface_file)
 
     elif project_id_src is not None:
-        try:
-            json_interface = cast(
-                List[Dict], kili.projects(project_id=project_id_src, disable_tqdm=True)
-            )[0]["jsonInterface"]
-        except:
-            # pylint: disable=raise-missing-from
-            raise ValueError(f"{project_id_src} is not recognized as a Kili project_id")
+        json_interface = services.get_project(kili, project_id_src, ["jsonInterface"])[
```

Comment:
Yes this could be another option to avoid having `services.get_project(kili, project_id_src, ["jsonInterface"])["jsonInterface"]`.

But isn't this a bit overkilled to avoid to access the `["jsonInterface"]`after ? Because there are also several cases where we want to get several fields.

Reply:
done